meson.build

```diff
@@ -152,7 +152,25 @@ if get_option('build_backends')
 
     elif get_option('openblas') and openblas_lib.found()
       add_project_arguments('-DUSE_OPENBLAS', language : 'cpp')
-      includes += include_directories(get_option('openblas_include'))
+
+      openblas_headers_found = false
```

Comment:
That's fine if that's the easiest way to implement that, but maybe there is another way.
I suggest you to show this code in #mesonbuild channel on irc.freenode.org, maybe they will suggest something.
Or I can do that myself tomorrow.